DownloadManager/Plugin.DownloadManager.iOS/UrlSessionDownloadDelegate.cs

```diff
@@ -82,8 +82,13 @@ public override void DidFinishDownloading (NSUrlSession session, NSUrlSessionDow
             var success = true;
             var destinationPathName = Controller.PathNameForDownloadedFile?.Invoke (file);
             if (destinationPathName != null) {
+                file.DestinationPathName = destinationPathName;
```

Comment:
Please set the path after the file has been moved. This way if a person only listens on a change on `DestinationPathName` his closure will be triggered when the file is available there.

Reply:
Good one. Will do.